src/EnergyPlus/PlantLoopEquip.cc

```diff
@@ -345,32 +344,9 @@ namespace PlantLoopEquip {
                 }
 
             } else if (EquipTypeNum == TypeOf_Chiller_Absorption) {
-                SimBLASTAbsorber(sim_component.TypeOf,
-                                 sim_component.Name,
-                                 EquipFlowCtrl,
-                                 LoopNum,
-                                 LoopSideNum,
-                                 EquipNum,
-                                 RunFlag,
-                                 FirstHVACIteration,
-                                 InitLoopEquip,
-                                 CurLoad,
-                                 MaxLoad,
-                                 MinLoad,
-                                 OptLoad,
-                                 GetCompSizFac,
-                                 SizingFac,
-                                 TempCondInDesign);
-                if (InitLoopEquip) {
-                    sim_component.MaxLoad = MaxLoad;
-                    sim_component.MinLoad = MinLoad;
-                    sim_component.OptLoad = OptLoad;
-                    sim_component.CompNum = EquipNum;
-                    sim_component.TempDesCondIn = TempCondInDesign;
-                }
-                if (GetCompSizFac) {
-                    sim_component.SizFac = SizingFac;
-                }
+
+                dynamic_cast<ChillerAbsorption::BLASTAbsorberSpecs*> (sim_component.compPtr)->EquipFlowCtrl = EquipFlowCtrl;
```

Comment:
It might be better to just set this inside the function instead.  There are examples of this but I can't think of one at the moment.  Basically, in the oneTimeInit function, or the simulate function, get the flow control value off the plant structure:

```
this->EquipFlowCtrl = DataPlant::PlantLoop(componentLocation.LoopNum).LoopSide(...blah...    .EquipFlowCtrl;
```

This avoids the dynamic cast.

Reply:
@Myoldmopar fixed, and develop is pulled in.